package.json

```diff
@@ -24,7 +23,7 @@
     "lint": "tsc --noEmit && prettier --check . && eslint .",
     "prepare": "husky install",
     "strings:extract": "lit-localize extract",
-    "strings:build": "lit-localize build && prettier --write src/locales"
+    "strings:build": "lit-localize build && sed -e 's!@lit/localize!../str!' -i '' src/i18n/strings/*.ts && prettier --write src/i18n"
```

Comment:
Do we get this from a dev dependency now?

Reply:
You mean `sed`? It's included in every Unix-like OS. (Also this command is only run in local dev, not in CI, so it only needs to work on macOS.)

To be clear, the command is making this change: [src/i18n/strings/es.ts](https://github.com/rewiringamerica/embed.rewiringamerica.org/pull/126/files#diff-056f9702ed61bdcb40e260d5b118d5971d61a4088e7e00c4479b92c25e870707)

Reply:
No I meant the `lit-localize` command - I see there's still `lit/localize-tools` in dev deps, which is what I was checking. From the diff I could only see the removal of the runtime package.